draft-ietf-tls-esni.md

```diff
@@ -1156,7 +1209,7 @@ IANA is requested to create the following two entries in the existing registry
 for ExtensionType (defined in {{!RFC8446}}):
 
 1. encrypted_client_hello(0xff02), with "TLS 1.3" column values being set to
-   "CH, EE", and "Recommended" column being set to "Yes".
+   "CH, SH, EE", and "Recommended" column being set to "Yes".
```

Comment:
Does this design variant ever send the ECH extension in the ServerHello?  Or is it only in the Encrypted Extensions?

Reply:
Only in EE.

Reply:
```suggestion
   "CH, EE", and "Recommended" column being set to "Yes".
```